posthog/__init__.py

```diff
@@ -19,6 +19,7 @@
 poll_interval = 30  # type: int
 disable_geoip = True  # type: bool
 feature_flags_request_timeout_seconds = 3  # type: int
+alpha_enable_exception_autocapture = False  # type: bool
```

Comment:
what if we add a comment that is in alpha? otherwise, we'd need to rename it to `beta_*` and remove the prefix afterward (unnecessary breaking changes).

Reply:
hmm, we can make it not-breaking, but agree, makes sense to not need the changes at all. 👍